notebooks/xgboost-model/xgboostDmatrixExample.ipynb

```diff
@@ -19,44 +19,31 @@
     "- In the example below we show how to create an xgboost model and evaluate fairness using Cortex Certifai\n",
     "- Example can be worked locally by installing the dependencies listed below\n",
     "- dependencies\n",
-    "    - python>=3.6.2,<=3.7\n",
-    "    - scikit-learn=0.20.3\n",
+    "    - installed Certifai toolkit\n",
     "    - xgboost (`conda install -c conda-forge xgboost`)\n",
-    "    - numpy=1.16.2\n",
-    "    - pandas\n",
     "    - ipython\n",
     "    - matplotlib\n",
     "    - jupyter\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 34,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# neccessary imports\n",
+    "# necessary imports\n",
     "import xgboost as xgb\n",
     "from sklearn.metrics import accuracy_score\n",
     "import pandas as pd\n",
     "import numpy as np\n",
-    "from sklearn.model_selection import train_test_split"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 35,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# special import - \n",
-    "# for multiprocessing to work in a Notebook,  pickled classes must be in a separate package or notebook\n",
-    "# hence, the model encoder(s),decoder class has to be somewhere other than the current notebook\n",
-    "\n",
-    "import os\n",
-    "import sys\n",
-    "sys.path.insert(0, os.path.abspath(os.path.join('.')))\n",
-    "from cat_encoder import CatEncoder"
+    "from sklearn.model_selection import train_test_split\n",
+    "# imports for building certifai scan\n",
+    "from certifai.scanner.builder import (CertifaiScanBuilder, CertifaiPredictorWrapper, CertifaiModel, CertifaiModelMetric,\n",
+    "                                      CertifaiDataset, CertifaiGroupingFeature, CertifaiDatasetSource,\n",
+    "                                      CertifaiPredictionTask, CertifaiTaskOutcomes, CertifaiOutcomeValue)\n",
+    "from certifai.scanner.report_utils import scores, construct_scores_dataframe\n",
+    "from certifai.common.utils.encoding import CatEncoder"
```

Comment:
even tho this is imported from common lib but in cell 5 it again imports the encoder from cat_encoder file. need to remove that
```
# create encoder for categorical columns
from cat_encoder import CatEncoder
cat_columns = [
   'workclass', 
   'education', 
   'marital-status', 
   'occupation', 
   'relationship',
   'race',
   'gender',
   'native-country'
           ]
encoder = CatEncoder(cat_columns, X_train)```

Reply:
need to delete the file `cat_encoder.py` and remove the re-import in `cell5`